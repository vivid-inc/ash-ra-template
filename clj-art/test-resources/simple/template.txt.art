<%

(def spring-flowers '[dandelions
                      crocuses
                      daffodils
                      tulips
                      hyacinths
                      bluebells])

%>Rejoice, for Spring brings such warmth to our hearts with <%= (clojure.string/join ", " spring-flowers) %> and more!
