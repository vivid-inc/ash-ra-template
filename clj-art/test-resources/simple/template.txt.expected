Rejoice, for Spring brings such warmth to our hearts with dandelions, crocuses, daffodils, tulips, hyacinths, bluebells and more!
